packages/ckeditor5-autoformat/tests/blockautoformatediting.js

```diff
@@ -167,6 +167,18 @@ describe( 'blockAutoformatEditing', () => {
 			sinon.assert.notCalled( spy );
 		} );
 
+		it( 'should not call callback when typing in the middle of block text', () => {
+			const spy = testUtils.sinon.spy();
```

Comment:
Yea... the main reason why I went with `const spy = testUtils.sinon.spy();` is because I duplicated another test from this suite.

Given that the test is fairly simple, uses only a single spy, I think it's ok to leave it unnamed. And I agree that it would stand out compared to other TCs in this suite.

WDYT @tomalec?